src/Text.mo

```diff
@@ -77,6 +114,12 @@ module {
   public func greaterOrEqual(t1 : Text, t2 : Text) : Bool { t1 >= t2 };
 
   /// Returns the order of `t1` and `t2`.
+  ///
+  /// ```motoko include=import
+  /// assert Text.compare("abc", "abc") == #equal;
+  /// assert Text.compare("abc", "def") == #less;
+  /// assert Text.compare("abc", "ABC") == #greater;
```

Comment:
I feel like assertions make the example harder to read, as opposed to having the interpreter display the value. What do you think?
```suggestion
  /// Text.compare("abc", "ABC")
```

Reply:
Trying to think of a way to show all of them at the same time without too much extra logic. Maybe we could have a few assertions and then an additional line with a comment saying "Try it yourself"?

Another consideration is that these will eventually appear as Markdown in the VS Code extension, so that was the main reference point I was using for writing these examples.

Reply:
Alternatively, you can just keep one of the three statements

Reply:
The reasoning for keeping all three is that it might be unclear to a new developer what output to expect in each of these situations (since the polymorphic variants, ASCII capitalization, and argument order could each individually be confusing without a few reference points). However it's probably fine just to show the last one and let people type in different Text values. Either option seems good to me.

Reply:
Right okay, I think you're right. It's good to see all cases for variants